recipes/polyglotdb/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "polyglotdb" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 89d2b4a19595a7bc14d1fba124afc63454baa9678275cd093d091d0430fc36c2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Would it be possible to convert the `scripts` tag upstream to `entry points`?
Then this could be noarch.

Ref: https://github.com/MontrealCorpusTools/PolyglotDB/blob/f021a403566388a9e13c76c2d727979baf4264e1/setup.py#L102

---

https://github.com/MontrealCorpusTools/PolyglotDB/pull/167